docs/UserGuide.md

```diff
@@ -407,6 +409,31 @@ Duplicate handling:
 
 ### Emergency contact name
 
+Format: `emergencyName` where `emergencyName` is a sequence of characters corresponding to a student's name
```

Comment:
Would it be clearer to just write "A sequence of alphanumeric corresponding to a student's name"? I'm not sure about this so up to you as well.